app/grandchallenge/cases/widgets.py

```diff
@@ -0,0 +1,114 @@
+from django.core.exceptions import ValidationError
+from django.forms import (
+    HiddenInput,
+    ModelChoiceField,
+    ModelMultipleChoiceField,
+    MultiValueField,
+    MultiWidget,
+)
+from django.forms.widgets import ChoiceWidget
+from django.utils.datastructures import MultiValueDictKeyError
+
+from grandchallenge.cases.models import Image
+from grandchallenge.cases.utils import WidgetChoices
+from grandchallenge.uploads.widgets import UserUploadMultipleWidget
+
+
+class ImageSearchMixin(ChoiceWidget):

```

Comment:
Why make this a Mixin and not just put it all in `ImageSearchWidget`?

Reply:
Good point, left over from an earlier implementation where that made sense, just forgot to refactor.